tests/webview/ui/test_content.py

```diff
@@ -342,26 +342,25 @@ def test_share_links_displayed(webview_base_url, selenium):
     home = Home(selenium, webview_base_url).open()
 
     # WHEN a book is clicked
-    book = home.featured_books.openstax_list[1]
+    book = home.featured_books.openstax_list[59]
     content = book.click_book_cover()
 
     # THEN social share links have the expected urls
     current_url = selenium.current_url
     normalized_title = content.title.replace(" ", "%20")
-    share = content.share
 
     expected_facebook_url = "https://facebook.com/sharer/sharer.php?u={url}".format(url=current_url)
-    assert share.facebook_share_url == expected_facebook_url
+    assert "https://facebook.com/sharer/sharer" in expected_facebook_url
```

Comment:
I'm a bit confused by these assertion changes since they now don't use `share` from L351 at all. Isn't this just checking against the string in L352 now? (e.g. won't it always pass no matter what)

Reply:
omg... this is wrong. thanks. reworking this part

Reply:
@rnathuji I changed the assertions in 'def test_share_links_displayed'